torch/fx/experimental/sym_node.py

```diff
@@ -181,6 +192,39 @@ def is_nested_int(self):
             and self._hint.node.is_nested_int()
         )
 
+    def nested_int(self):
+        # Not really used for comparison. Only used to get at the data in
+        # the equiv set. The equiv set for symbolic is different than that
+        # of the hint.
+        assert self.is_nested_int()
+        return self._nested_int
+
+    def nested_int_vec(self):
+        assert self._nested_int_vec is not None
+        return self._nested_int_vec
+
+    def clone_nested_int_with_new_vec(self, equiv_set, vec):
```

Comment:
Not necessarily the same equiv set. There are actually two situations where this is used:
1) when AOTAutograd creates FakeTensor grad_outputs, it recursively calls .view_as(self) on every inner dense tensor via a transform_subclass. In this case, yes, we want to add the new nested int to the same equiv set.
2) when AOTAutograd does functionalization, it calls transform_subclass to swap all FakeTensors with FunctionalTensors, we want a different equiv set in this case because we don't want the FunctionalTensors to share the same equiv set cache as FakeTensors.